software-engineering-policies/InfrastructureAsCode/terraform.md

```diff
@@ -0,0 +1,38 @@
+# Infrastructure as Code: Terraform
+
+Terraform is our tool of choice, and should be considered in the first instance.
+
+Most standards and practises at the UKHO is covered in the Training courses we provide around the use of Terraform.
```

Comment:
Add some more detail on this section which signposts who to contact if you need this training and where to find the pre-recorded videos.